libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Actions/InitProperty.cs

```diff
@@ -1,103 +0,0 @@
-﻿// Licensed under the MIT License.
-// Copyright (c) Microsoft Corporation. All rights reserved.
-
-using System;
-using System.Runtime.CompilerServices;
-using System.Threading;
-using System.Threading.Tasks;
-using Microsoft.Bot.Expressions;
-using Newtonsoft.Json;
-using Newtonsoft.Json.Linq;
-
-namespace Microsoft.Bot.Builder.Dialogs.Adaptive.Actions
-{
-    /// <summary>
-    /// Sets a property with the result of evaluating a value expression.
-    /// </summary>
-    public class InitProperty : Dialog
```

Comment:
>InitProperty [](start = 17, length = 12)

the InitProperty.schema file should be deleted too, or it will be re-include on next bf merge